packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfApplication-CSharp/.template.config/template.json

```diff
@@ -59,7 +59,8 @@
     {
       "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
       "path": "MainWindow.xaml.cs"
-    }
+    },
+    { "path": "MainWindow.xaml" }
```

Comment:
There is no condition here similar to `MainWindow.xaml.cs`. Can you help me understand whether this was intentional, and if so, why we'd want `MainWindow.xaml` included in `primaryOutputs` when `HostIdentifer=="dotnetcli*"`, for e.g? 

I notice that `postActions` is set up not to react to this addition even when `Hostidentifer=="dotnetcli*"`, so perhaps its not a big problem. We should probably still maintain the right conditions if its easy to do so.